components/output/mcp4461.rst

```diff
@@ -0,0 +1,188 @@
+    output:
+    - platform: mcp4461
+      id: digipot_channel_0
+      mcp4461_id: mcp4461_output
+      channel: A
+      initial_value: 0.128 // always initialize volatile wiper 0 with wiper @ medium resistance range on start
+    - platform: mcp4461
+      id: digipot_channel_1
+      mcp4461_id: mcp4461_output
+      channel: B
+    - platform: mcp4461
+      id: digipot_channel_2
+      mcp4461_id: mcp4461_output
+      channel: C
+    - platform: mcp4461
+      id: digipot_channel_3
+      mcp4461_id: mcp4461_output
+      channel: D
+    - platform: mcp4461
+      id: digipot_channel_4
+      mcp4461_id: mcp4461_output
+      channel: E
+    - platform: mcp4461
+      id: digipot_channel_5
+      mcp4461_id: mcp4461_output
+      channel: F
+    - platform: mcp4461
+      id: digipot_channel_6
+      mcp4461_id: mcp4461_output
+      channel: G
+    - platform: mcp4461
+      id: digipot_channel_7
+      mcp4461_id: mcp4461_output
+      channel: H
+
+    on_...:
+      then:
+        - output.set_level:
+            id: digipot_channel_0
+            level: !lambda return static_cast<float>(x / 256); // using a uint16_t in range 0-256
+                   // !lambda return x; - if x is float in range 0.0-1.0,
+                   // will be multiplied with 256 internally to get integer taps value
+
+    
+    interval:
+      // read current hardware wiper state to variable without updating output-channel state
+      - interval: 5s
+      // update wiper state every 10s (potentiometer usage)
+      - interval: 10s
+          then:
+            - lambda: |-
+                float current_state;
+                current_state = id(digipot_wiper_0).read_state();
+                return;
+
+    button:
+      - platform: template
+        id: increment_wiper_0_button
+        name: "Increment wiper 0"
+        on_press:
+          then:
+            - lambda: |-
+                id(digipot_channel_0).increase_wiper(); // increase taps by one for wiper 0
+                return;
+
+      - platform: template
+        id: enable_wiper_0_button
+        name: "Enable wiper 0"
+        on_press:
+          then:
+            - lambda: |-
+                id(digipot_channel_0).turn_on(); // same as id(digipot_wiper_0).enable_wiper() and id(digipot_wiper_0).enable_terminal('h')
+                return;
+
+
+      - platform: template
+        id: enable_wiper_0_terminal_a_button
+        name: "Enable wiper 0 terminal A"
+        on_press:
+          then:
+            - lambda: |-
+                id(digipot_channel_0).enable_terminal('a'); // available terminals are { 'h', 'a', 'b', 'w' }
+                return;
+
+      - platform: template
+        id: disable_wiper_0_terminal_a_button
+        name: "Disable wiper 0 terminal A"
+        on_press:
+          then:
+            - lambda: |-
+                id(digipot_channel_0).disable_terminal('a'); // available terminals are { 'h', 'a', 'b', 'w' }
+                return;
```

Comment:
As above.
```suggestion
```
If you want to write complete examples with details, they need to go into a cookbook.